python/cugraph/bsp/traversal/bfs_bsp.py

```diff
@@ -92,7 +92,7 @@ def bfs_df_pregel(_df, start, src_col='src', dst_col='dst', copy_data=True):
         # we do not want to hop to a vertex that has already been seen
         coo_data = coo_data.merge(frontier, on=['dst'], how='left')
         coo_data = coo_data[coo_data.distance.isnull()]
-        coo_data.drop_column('distance')
+        coo_data.drop('distance')
```

Comment:
this is an out-of-place call, so column is not removed from the dataframe

Reply:
fixed